docs/api/api/api.txt

```diff
@@ -880,9 +879,20 @@ Parameters:
   onlyissues: used to limit to issues (for diff commands)
   setrelease: define a specific release tag when used with "release" command. Setting it to "-" strips
               the release string.
-  withvalidate: activate sha validation code
   withvrev: copy also the vrev counter of the revision
 
+POST /source/<project>/<package>?cmd=commitfilelist
+
+  commit defined files in upload revision
```

Comment:
This sounds like this API call would result in a commit being created, but I thought that wasn't the case here?

Reply:
it creates a commit, yes

Reply:
Ah, in that case we should also mention the parameter `comment` here.

Reply:
right, added